### ISSUE TITLE
Mobile: Countdown doesn't go away

### ISSUE THREAD

**Issue by alykat** [nprapps/elections14#1241]
The countdown resolves to this:

![img_2958](https://cloud.githubusercontent.com/assets/491318/4885970/7c6d699a-6378-11e4-9d1b-e20c91542e94.png)

The audio plays, but the countdown doesn't go away and the board doesn't load.

**Comment by alykat** [comment]
Seems to be an error on touch devices looking for:

```
    if (IS_TOUCH && Modernizr.mq('(orientation: portrait)')) {
```

**Comment by alykat** [comment]
@eads fixed in 7d669091e3b221039a0d15191b29b981457af3e3
